src/pages/workspace/companyCards/addNew/StatementCloseDateStep.tsx

```diff
@@ -17,29 +18,35 @@ type StatementCloseDateStepProps = {
 
 function StatementCloseDateStep({policyID}: StatementCloseDateStepProps) {
     const {translate} = useLocalize();
-
+    const {isBetaEnabled} = usePermissions();
     const [addNewCard] = useOnyx(ONYXKEYS.ADD_NEW_COMPANY_CARD, {canBeMissing: false});
     const [lastSelectedFeed] = useOnyx(`${ONYXKEYS.COLLECTION.LAST_SELECTED_FEED}${policyID}`, {canBeMissing: true});
-
     const [cardFeeds] = useCardFeeds(policyID);
 
+    const isPlaid = isBetaEnabled(CONST.BETAS.PLAID_COMPANY_CARDS) && !!addNewCard?.data?.publicToken;
```

Comment:
We released this beta to everyone, so no need to check for it here

Reply:
I copied this logic from `src/pages/workspace/companyCards/BankConnection/index.tsx` I think it's okay to keep it for consistency. Ideally I think there should be a PR that removes the beta.

Reply:
I think there will be a separate PR to remove this logic. We could also remove this instance in this PR, but NAB.